plugins/chat/spec/models/chat/thread_spec.rb

```diff
@@ -9,17 +9,23 @@
     fab!(:thread_2) { Fabricate(:chat_thread, channel: channel) }
     fab!(:thread_3) { Fabricate(:chat_thread, channel: channel) }
 
-    before do
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_1)
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_1)
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_1)
-
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_2)
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_2)
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_2)
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_2)
-
-      Fabricate(:chat_message, chat_channel: channel, thread: thread_3)
+    fab!(:thread_1_message_1) { Fabricate(:chat_message, chat_channel: channel, thread: thread_1) }
+    fab!(:thread_1_message_2) { Fabricate(:chat_message, chat_channel: channel, thread: thread_1) }
+    fab!(:thread_1_message_3) { Fabricate(:chat_message, chat_channel: channel, thread: thread_1) }
+
+    fab!(:thread_2_message_1) { Fabricate(:chat_message, chat_channel: channel, thread: thread_2) }
+    fab!(:thread_2_message_2) { Fabricate(:chat_message, chat_channel: channel, thread: thread_2) }
+    fab!(:thread_2_message_3) { Fabricate(:chat_message, chat_channel: channel, thread: thread_2) }
+    fab!(:thread_2_message_4) { Fabricate(:chat_message, chat_channel: channel, thread: thread_2) }
+
+    fab!(:thread_3_message_1) { Fabricate(:chat_message, chat_channel: channel, thread: thread_3) }
+
+    it "supports custom fields" do
```

Comment:
We’re in the `.ensure_consistency!` method spec here, so this should be taken outside the current `describe` block.

Reply:
moved :+1: